src/screens/pos-actions/pos-action-quick-delegate/quick-delegate-select-validator/quick-delegate-select-validator.tsx

```diff
@@ -98,14 +92,8 @@ export class QuickDelegateSelectValidatorComponent extends React.Component<
         super(props);
 
         this.state = {
-            nrValidators:
-                props.validators.length < defaultNumberOfValidators
-                    ? props.validators.length
-                    : defaultNumberOfValidators,
-            validatorsList:
-                props.validators.length > defaultNumberOfValidators
-                    ? props.validators.slice(0, defaultNumberOfValidators)
-                    : props.validators
+            nrValidators: props.validators.length,
+            validatorsList: props.validators
```

Comment:
Also this can be removed since it's not used anymore. Use props.validators

Reply:
This is needed in order to perform actions on the list ex: selecting a validator.

Reply:
Yes indeed, true